packages/performance-tests/test/TransformerRegression.test.ts

```diff
@@ -38,6 +40,22 @@ const testCasesMap = new Map([
 const loggerCategory = "Transformer Performance Regression Tests";
 const outputDir = path.join(__dirname, ".output");
 
+const loadTransformers = async () => {
+  const moduleNames = process.env.EXTRA_TRANSFORMERS?.split(',').map((name) => name.trim()) ?? [];
+  const transformerModules = [ProgressTransformer];
+
+  for (const moduleName of moduleNames) {
+    try {
+      const module = (await import(moduleName)).default;
+      transformerModules.push(module.default);
```

Comment:
do we really need two levels of default? I encountered this before... is it due to esmodule interop?